### PR TITLE
Update choose cluster docs

### PR DESCRIPTION
**Pull request by danpaul000**
#### Problem
No information about connecting to mainnet beta in our CLI docs

#### Summary of Changes
Add it and clean up Devnet/Testnet information

### CODE REVIEW

**Comment by danpaul000** [comment]
@mvines anything to add/edit in how I describe the difference between our 3 clusters?

File: docs/src/cli/choose-a-cluster.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# Connecting to a Cluster
+
+Solana maintains several different clusters with different purposes.  Each
+cluster features a Solana-owned validator that serves as an entrypoint to the
```
Comment (mvines) [inline docs/src/cli/choose-a-cluster.md:4]:
```suggestion
cluster features a Solana-run gossip node that serves as an entrypoint to the
```

File: docs/src/cli/choose-a-cluster.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+test drive, as a user, token holder, app developer, or validator.
+* Application developers should target Devnet.
+* Potential validators should first target Devnet.
+* Key differences between Devnet and Mainnet Beta:
+  * Devnet tokens are **not real**
+  * Devnet includes a token faucet for airdrops for application testing
+  * Devnet may be subject to ledger resets
+  * Devnet typically runs a newer software version than Mainnet Beta
+  * Devnet may be maintained by different validators than Mainnet Beta
+ * Devnet is upgraded automatically on the last Monday of each month.
```
Comment (mvines) [inline docs/src/cli/choose-a-cluster.md:36]: IMO remove, this isn't too helpful and is not totally accurate.    " Devnet typically runs a newer software version than Mainnet Beta" is sufficient IMO
```suggestion

```

File: docs/src/cli/choose-a-cluster.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+ * Entrypoint URL for Devnet: `https://devnet.solana.com`
```
Comment (mvines) [inline docs/src/cli/choose-a-cluster.md:37]:
```suggestion
 * Gossip entrypoint for Devnet: `devnet.solana.com:8001`
 * RPC URL for Devnet: `https://devnet.solana.com`
```

File: docs/src/cli/choose-a-cluster.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+behavior.
+* [Tour de SOL](../tour-de-sol/README.md) initiative runs on Testnet, where we
+encourage malicious behavior and attacks on the network to help us find and
+squash bugs or network vulnerabilities.
+* Testnet tokens are **not real**
+* Testnet may be subject to ledger resets.
+* Testnet typically runs a newer software release than both Devnet and
+Mainnet Beta
+* Testnet may be maintained by different validators than Mainnet Beta
+* Entrypoint URL for Testnet: `https://testnet.solana.com`
```
Comment (mvines) [inline docs/src/cli/choose-a-cluster.md:51]:
```suggestion
* Gossip entrypoint for Testnet: `testnet.solana.com:8001`
* RPC URL for Testnet: `https://testnet.solana.com`
```

File: docs/src/cli/choose-a-cluster.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+#### Mainnet Beta
+A permissionless, persistent cluster for early token holders and launch partners.
+No smart contracts or inflation.
+ * Tokens that are issued on Mainnet Beta are **real** SOL
+ * If you have paid money to purchase/be issued tokens, such as through our
+ CoinList auction, these tokens will be transferred on Mainnet Beta.
+   * Note: If you are using a non-command-line wallet such as
+   [Trust Wallet](../wallet/trust-wallet.md),
+   the wallet will always be connecting to Mainnet Beta.
+ * Entrypoint URL for Mainnet Beta: `https://api.mainnet-beta.solana.com`
```
Comment (mvines) [inline docs/src/cli/choose-a-cluster.md:23]:
```suggestion
 * Gossip entrypoint for Mainnet Beta: `mainnet-beta.solana.com:8001`
 * RPC URL for Mainnet Beta: `https://api.mainnet-beta.solana.com`
```

**Pull request merged**